tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -13,11 +13,13 @@
 import tensorflow_hub as hub
 # noinspection PyUnresolvedReferences
 import tensorflow_text  # do not delete, needed for text models
+from openvino.runtime import Core
```

Comment:
```suggestion
from openvino import Core
```
let use current recommended API